test/render/test-cases.js

```diff
@@ -1747,5 +1771,85 @@ export const TEST_CASES = [
       })
     ],
     goldenImage: './test/render/golden-images/s2-layer-l2.png'
+  },
+  {
+    name: 'simple-mesh-layer',
+    viewState: {
+      latitude: 37.75,
+      longitude: -122.45,
+      zoom: 11.5,
+      pitch: 0,
+      bearing: 0
+    },
+    layers: [
+      new SimpleMeshLayer({
+        id: 'simple-mesh-layer',
+        data: dataSamples.points.slice(0, 10),
+        mesh: cube,
+        sizeScale: 100,
+        modelMatrix: new Matrix4().rotateX((-45 / 180) * Math.PI),
+        coordinateSystem: COORDINATE_SYSTEM.DEFAULT,
+        getPosition: d => d.COORDINATES,
+        getColor: [0, 255, 255, 125],
+        getTransformMatrix: meshLayerInstanceMatrix
+      })
+    ],
+    goldenImage: './test/render/golden-images/simple-mesh-layer.png'
+  },
+  {
+    name: 'simple-mesh-layer-modelmatrix',
+    viewState: {
+      target: [0, 0, 0],
+      rotationX: 0,
+      rotationOrbit: 0,
+      orbitAxis: 'Y',
+      fov: 30,
+      zoom: -1.5
+    },
+    views: [
+      new OrbitView({
+        near: 0.1,
+        far: 2
+      })
+    ],
+    layers: [
+      new SimpleMeshLayer({
+        id: 'simple-mesh-layer-cartesian',
+        data: SAMPLE_DATA,
+        mesh: cube,
+        sizeScale: 10,
+        coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
+        modelMatrix: new Matrix4().rotateX((-45 / 180) * Math.PI),
+        getPosition: d => d.position,
+        getColor: d => d.color,
+        getOrientation: d => d.orientation
+      })
+    ],
+    goldenImage: './test/render/golden-images/simple-mesh-layer-cartesian.png'
+  },
+  {
+    name: 'simple-mesh-layer-meter-offsets',
```

Comment:
I think it's more important to test the LNGLAT use case

Reply:
I have one test for `COORDINATES.DEFAULT`, which is lnglat